tests/phpunit/integration/Modules/Analytics_4Test.php

```diff
@@ -4414,47 +4411,6 @@ function ( Request $request ) use ( $raw_audiences, $property_id ) {
 		);
 	}
 
-	/**
-	 * @dataProvider data_access_token
-	 */
-	public function test_site_kit_audiences_returned_in_debug_fields( $access_token ) {
```

Comment:
This test should be restored, as it's testing some logic which isn't covered elsewhere. Again, I'll go ahead and restore this myself.